x/exercise.rb

```diff
@@ -0,0 +1,7 @@
+module X
+  module Exercise
+    def self.exists?(language, slug)
+      Xapi.request('tracks', language, slug).status != 404
```

Comment:
What would be the best way to test this? All of the existing tests stub out this method, and nothing tests that it actually talks to the correct endpoint / has the correct behavior.